index.html

```diff
@@ -2182,29 +2182,33 @@ <h2 id="state-transitions" class="informative">
           <var>request</var>.<a>[[\options]]</a> is true, then copy the <code>
             shippingAddress</code> attribute of <var>request</var> to the
             <a data-lt="PaymentResponse.shippingAddress">shippingAddress</a>
-            attribute of <var>response</var>.
+            attribute of <var>response</var>; or to <code>null</code> if none
```

Comment:
Nit: comma, not semicolon, since the second clause is not a standalone sentence. http://data.grammarbook.com/blog/commas/how-to-punctuate-between-sentences-using-commas-semicolons-and-colons/